docs/user/general/datatypes.rst

```diff
@@ -456,3 +458,17 @@ A boolean can be represented by constant value ``TRUE`` or ``FALSE``. Besides, c
     |--------+---------+---------------------------+----------------------------|
     | True   | False   | True                      | False                      |
     +--------+---------+---------------------------+----------------------------+
+
+Geopoint Data Types
+==================
+
+A geopoint has a latitude and a longitude property. Although OpenSearch `supports multiple formats <https://opensearch.org/docs/2.3/opensearch/supported-field-types/geo-point/>`_, the SQL plugin currently only supports the format :code:`{"lat": number, "lon": number}`. The geopoint object can be queried or lat and lon can be specified using dot notation. For example, ::
+
+    os> SELECT
+    ...  geo_point_object, geo_point_object.lat, geo_point_object.lon;
```

Comment:
Why do you have the `SELECT` on a different line to the rest of the query rather than having something like:
```os> SELECT geo_point_object, geo_point_object.lat, geo_point_object.lon;```